ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/batch/BatchedSubmissionValidator.scala

```diff
@@ -402,41 +401,5 @@ class BatchedSubmissionValidator[CommitResult] private[validator] (
     }
   }
 
-  private[batch] object Metrics {
-    private val Prefix = MetricName.DAML :+ "pkvutils" :+ "batch_validator"
-
-    val validateAndCommit: Timer =
-      metrics.registry.timer(Prefix :+ "validate_and_commit")
-
-    val openEnvelope: Timer =
-      metrics.registry.timer(Prefix :+ "open_envelope")
-
-    val batchSizes: Histogram =
-      metrics.registry.histogram(Prefix :+ "batch_sizes")
-
-    val receivedBatchSubmissionBytes: Histogram =
-      metrics.registry.histogram(Prefix :+ "received_batch_submission_bytes")
-
-    val receivedSubmissionBytes: Histogram =
-      metrics.registry.histogram(Prefix :+ "received_submission_bytes")
-
-    // Metrics for each stage. We both time and maintain a counter for each running stage.
-    // With the counter we can track how many submissions we're processing in parallel.
-    // We don't have metrics for stages 4 and 5 since they're trivial (collect and sort).
-
-    val decode: Timer = metrics.registry.timer(Prefix :+ "decode")
-    val decodeRunning: Counter = metrics.registry.counter(Prefix :+ "decode_running")
-
-    val fetchInputs: Timer = metrics.registry.timer(Prefix :+ "fetch_inputs")
-    val fetchInputsRunning: Counter = metrics.registry.counter(Prefix :+ "fetch_inputs_running")
-
-    val validate: Timer = metrics.registry.timer(Prefix :+ "validate")
-    val validateRunning: Counter = metrics.registry.counter(Prefix :+ "validate_running")
-
-    val detectConflicts: Timer =
-      metrics.registry.timer(Prefix :+ "detect_conflicts")
-
-    val commit: Timer = metrics.registry.timer(Prefix :+ "commit")
-    val commitRunning: Counter = metrics.registry.counter(Prefix :+ "commit_running")
-  }
+  private[this] val metrics = damlMetrics.daml.kvutils.submission.validator
```

Comment:
Can you move this to the top, please? It's odd to see `val`s referred to before they're used.

Reply:
Moved there.